test/OptionsTest.php

```diff
@@ -8,18 +8,21 @@
 
 class OptionsTest extends TestCase
 {
-  public function test() {
-      $options = new Options();
-      $options->add("hello", "goodbye");
-      $options->add("hola", "chao");
-      $options->add("multi", (new Sequence())->add('adieu')->add('shalom'));
+    public function test()
+    {
+        $options = new Options();
+        $options->add("hello", "goodbye");
+        $options->add("hola", "chao");
+        $options->add("multi", (new Sequence())->add('adieu')->add('shalom'));
 
-      $this->assertEquals(json_encode($options->options()), json_encode(["hello", "hola", "multi"]));
-      $this->assertEquals($options->return("hello"), "goodbye");
-      $this->assertEquals($options->return("hola"), "chao");
-      $this->assertEquals($options->return("multi"), "adieu");
-      $this->assertEquals($options->return("multi"), "shalom");
-      $this->assertEquals($options->return("multi"), "shalom");
-      $this->assertNull($options->return('not-an-option'));
-  }
-}
\ No newline at end of file
+        $this->assertEquals(json_encode($options->options()), json_encode(["hello", "hola", "multi"]));
+        $this->assertEquals($options->return("hello"), "goodbye");
+        $this->assertEquals($options->return("hola"), "chao");
+        $this->assertEquals($options->return("multi"), "adieu");
+        $this->assertEquals($options->return("multi"), "shalom");
+        $this->assertEquals($options->return("multi"), "shalom");
+        $this->assertNull($options->return('not-an-option'));
+        $options->index(2);
```

Comment:
This is a dumb test, and should have an equivalent over on the ChainTest side that actually tests the functionality.